Assets/MixedRealityToolkit.Examples/Demos/Diagnostics/Scenes/DiagnosticsDemo.unity

```diff
@@ -50,7 +50,6 @@ LightmapSettings:
     m_BounceScale: 1
```

Comment:
This is great we have an example scene! In the future this could be a nice place to demo some examples of what could cause performance to decline or fluctuate (with our various diagnostic tools pointing out the problem(s)).